BricksAndHearts.UnitTests/ServiceTests/Api/ApiServiceTestsBase.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BricksAndHearts.Auth;
+using BricksAndHearts.Services;
+using FakeItEasy;
+using Microsoft.Extensions.Options;
+
+namespace BricksAndHearts.UnitTests.ServiceTests.Api;
+
+public class ApiServiceTestsBase
```

Comment:
Don't feel like you need to split this into a separate file, fine to merge it all into ApiServiceTests

Reply:
Ok I've merged it all into one file.